src/components/WalletCongresSenate/Motion/index.js

```diff
@@ -25,6 +26,8 @@ export default function Motion({
 
   const isClosable = voting.ayes.length >= threshold;
 
+  const isClosableNaye = voting.nays.length >= 50;
```

Comment:
It should be something like `voting.nays.length > number_of_congressmen - threshold`.

Reply:
yes i just take `has more than 50 nay votes` from task, too literally

Reply:
percent xD